src/hangman/models/game.model.ts

```diff
@@ -1,4 +1,6 @@
 import { AggregateRoot } from '@nestjs/cqrs';
```

Comment:
i don't think the aggregate should be a 'model' in a 'models' folder. Models to me are ORM entities.

Reply:
Yep agree